### ISSUE TITLE
Size of icon at left and right of header is not similar?

### ISSUE THREAD

**Issue by isaaclem** [GeekyAnts/NativeBase#2205]
```
        <View>
          <Header style={headerContainerStyle}>
            <Left style={headerLeftStyle}>
              <Button transparent>
                <Icon name='ios-refresh' type='Ionicons' style={headerIconStyle} />
              </Button>
            </Left>
            <Body style={{ flexDirection: 'column' }}>
              <Title style={headerSmallText}>Jobs</Title>
              <Title style={headerSmallSubText}>Today, someStatus</Title>
            </Body>
            <Right style={headerLeftStyle}>
              <Button transparent>
                <Icon name='ios-refresh' type='Ionicons' style={headerIconStyle} />
              </Button>
            </Right>
          </Header>
          <View>
            <Text>JobsScreen Component</Text>
          </View>
        </View>
```
## react-native, react and native-base version
```
"native-base": "^2.7.2",
"react": "16.4.1",
"react-native": "0.56.0",
```
## Expected behaviour
Refresh button for left and right to be the same size

## Actual behaviour
![screen shot 2018-08-13 at 9 02 36 pm](https://user-images.githubusercontent.com/32945572/44033268-7916aa5a-9f3c-11e8-9785-29a61ee1d8d0.png)

Very obvious the icon on the left is much larger. They are sharing the exact same style, `headerLeftStyle`, which is only a `marginLeft: 10`

## Is the bug present in both iOS and Android or in any one of them?
Both ios and android behave the same

**Comment by SupriyaKalghatgi** [comment]
@isaaclem true. This is due to https://github.com/GeekyAnts/NativeBase/blob/master/src/theme/components/Header.js#L320
The condition values should be reversed

**Comment by SupriyaKalghatgi** [comment]
Fixed https://github.com/GeekyAnts/NativeBase/commit/502f523726e5cda8c1729ff34aa7d839bafe6844

**Comment by SupriyaKalghatgi** [comment]
Fixed with [2.8.0](https://github.com/GeekyAnts/NativeBase/releases/tag/v2.8.0)
